pagopa/pn-frontend
# Notification empty state for operator users - pn-5323

## Short description
Removed "Recapiti" link for PG operator users in the notification list empty state

## List of changes proposed in this pull request
- Added check on user role in MobileNotifications and DesktopNotifications components

## How to test
Run PG app pointing to UAT and login with GabrieleDAnnunzio -> MarcoTullioCicerone and check that the empty state doesn't have the "Recapiti" link
